### PR TITLE
Fix trk2dictionary affine bug for TCK tractograms

### PR DESCRIPTION
**Pull request by MarioOcampo**
Create `dictionary_mask.nii.gz` and `dictionary_tdi.nii.gz` with affine from `TCK_ref_image` for TCK tractograms.

This PR fixes the problem for TCK tractograms.
In the case of TRK, the problem persists if a `mask` or `peaks` file is not specified.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
 # Change Log
 All notable changes to COMMIT will be documented in this file.
 
-## [1.4.6] - 2021-03-25
+## [1.4.6] - 2021-04-23
```
Comment (daducci) [inline CHANGELOG.md:5]: Shall we merge this PR inside v1.5.0?

**Comment by daducci** [comment]
When you say "the problem persists" you mean that the identity matrix is used as affine? I see that you added a warning, well done!

File: CHANGELOG.md (hunk 1 above)
Comment (MarioOcampo) [reply CHANGELOG.md:5]: Maybe, depending on which PR is merged first.

**Comment by MarioOcampo** [comment]
Yes, for TRK tractograms, if there is no mask or peaks image specified,  it uses the identity as affine.

File: commit/trk2dictionary/trk2dictionary.pyx [hunk 2]
```diff
@@ -406,13 +406,18 @@ cpdef run( filename_tractogram=None, path_out=None, filename_peaks=None, filenam
...
-        affine = np.diag( [Px, Py, Pz, 1] )
-
+    if extension == ".trk":
+        if filename_mask is not None :
+            affine = niiMASK.affine if nibabel.__version__ >= '2.0.0' else niiMASK.get_affine()
+        elif filename_peaks is not None :
+            affine = niiPEAKS.affine if nibabel.__version__ >= '2.0.0' else niiPEAKS.get_affine()
+        else :
+            affine = np.diag( [Px, Py, Pz, 1] )
+            WARNING( 'dictionary_mask.nii.gz and dictionary_tdi.nii.gz will be created with a default affine' )
```
Comment (daducci) [inline commit/trk2dictionary/trk2dictionary.pyx:416]: Change to "Output maps will be created using a default affine, i.e., identity"

**Comment by daducci** [comment]
We should change the order of files that are searched for when looking for an affine, in the case of TCK. In fact, now, this order is different from the one done for TRK.

File: commit/trk2dictionary/trk2dictionary.pyx (hunk 2 above)
Comment (MarioOcampo) [reply commit/trk2dictionary/trk2dictionary.pyx:416]: It is not identity, It has the voxel size [Px, Py, Pz], I propose 
WARNING( 'Output maps will be created using a default affine, i.e., diagonal matrix [%0.2f, %0.2f, %0.2f, 1.]'%(Px, Py, Pz) )

Comment (daducci) [reply commit/trk2dictionary/trk2dictionary.pyx]: I think it is ok like it is

**Pull request closed without merging**